lib/game.rb

```diff
@@ -72,4 +78,43 @@ def diagonal_down_sort
     end
     padded_matrix.transpose.map {|array| array.compact}
   end
+
+  def game_menu
+    puts 'Welcome to Connect 4!! \n 
+      Enter p to Enter p to play the highest stakes game of your life. Enter q to wimp out and quit.'
+    ans = gets.chomp
+    if ans == "p"
+      @board = Board.new
+      turn_round
+    elsif ans == "q"
+      puts `clear`
+      puts 'GAME OVER'
+    else
+      puts `clear`
+      puts "#{ans} isn't a valid answer fool!!"
+      game_menu
+    end
+  end
+
+  def turn_round
+    until @game_win != nil
+      puts `clear`
+      puts "You chose #{@player_turns.last} and the computer chose #{@comp_turns.last}"
+      @board.render
+      puts "What column do you choose, intrepid player?"
+      player_turn(gets.chomp)
+      computer_turn
+    end
+    if @game_win == true
+      puts `clear`
+      @player_wins += 1
+      puts 'YOU WON! HOW COOL!'
+    elsif @game_win == false
+      puts `clear`
+      @comp_wins += 1
+      puts 'YOU LOST! YOU SUCK!'
+    end
+    puts "You've won #{@player_wins} times and the computer has won #{@comp_wins} times."
+
```

Comment:
Needs game_menu here

Reply:
Also, might be good to put in a separate method so win screen can standalone and then add 'press any key to continue'